app/presenters/result_set_presenter.rb

```diff
@@ -142,9 +142,11 @@ def fetch_signup_links
     links = {}
     links[:email_signup_link] = email_signup_link if email_signup_link.present?
     links[:feed_link] = feed_link if feed_link.present?
-    links[:margin_bottom] = 0 if email_signup_link.present? || feed_link.present?
-    links[:hide_heading] = true
-    links[:small_form] = true
+    if email_signup_link.present? || feed_link.present?
+      links[:margin_bottom] = 0
```

Comment:
I think the default margin-bottom for the component is 0 if it's not specified, so I'm not sure this line is needed.

Reply:
Good point 👍